agent/component-test/softwarecontaineragent_componenttest.cpp

```diff
@@ -105,7 +105,7 @@ class SoftwareContainerAgentTest: public ::testing::Test
 
         try {
             std::shared_ptr<SoftwareContainerFactory> factory = std::shared_ptr<SoftwareContainerFactory> (new SoftwareContainerFactory());
-            std::shared_ptr<ContainerUtilityInterface> utility = std::shared_ptr<ContainerUtilityInterface> (new ContainerUtilityInterface());
+            std::shared_ptr<ContainerUtilityInterface> utility = std::shared_ptr<ContainerUtilityInterface> (new ContainerUtilityInterface(config));
```

Comment:
to be concise could you use somethin like that instead 
`auto utility = std::make_shared<ContainerUtilityInterface>(config);`